spec/lib/promisifier.spec.js

```diff
@@ -23,4 +23,17 @@ describe("promisifier", () => {
         expect(promisifier(obj)).toBe(result);
         expect(promiseMock.promisifyAll).toHaveBeenCalledWith(obj);
     });
+    it("promisifies a function as well", () => {
+        var fn, result;
+
+        fn = () => {};
+        result = () => {};
+        spyOn(promiseMock, "promisify").andReturn(result);
```

Comment:
Do you want to use spyOn here when we've been using jasmine.createSpy most other places? Do we need to specify a standard for use?